code/datums/components/crafting/recipes.dm

```diff
@@ -13,6 +17,8 @@
 	var/always_available = TRUE //Set to FALSE if it needs to be learned first.
 	/// Additonal requirements text shown in UI
 	var/additional_req_text
+	///Required machines for the craft, set the assigned value of the typepath to CRAFTING_MACHINERY_CONSUME or CRAFTING_MACHINERY_USE
+	var/machinery
```

Comment:
```suggestion
	///Required machines for the craft, set the assigned value of the typepath to CRAFTING_MACHINERY_CONSUME or CRAFTING_MACHINERY_USE. Lazy associative list: type_path key -> flag value.
	var/list/machinery
```
Something like that? Or rephrase it differently if you think other way better.

Reply:
Makese sense!